consensus/src/marshal/mod.rs

```diff
@@ -40,16 +40,17 @@
 //! The actor uses a combination of prunable and immutable storage to store blocks and
 //! finalizations. Prunable storage is used to store data that is only needed for a short
 //! period of time, such as unverified blocks or notarizations. Immutable storage is used to
-//! store data that needs to be persisted indefinitely, such as finalized blocks. This allows
-//! the actor to keep its storage footprint small while still providing a full history of the
-//! chain.
+//! store data that needs to be persisted indefinitely, such as finalized blocks.
+//!
+//! Marshal will store all blocks after a configurable starting height onward. This allows for state
+//! sync from a specific height rather than from genesis.
 //!
 //! ## Limitations and Future Work
 //!
 //! - Only works with [crate::simplex] rather than general consensus.
 //! - Assumes at-most one notarization per view, incompatible with some consensus protocols.
-//! - No state sync supported. Will attempt to sync every block in the history of the chain.
-//! - Stores the entire history of the chain, which requires indefinite amounts of disk space.
+//! - Does not prune blocks that exist in the immutable archive below the sync floor. Although these
+//!   blocks are no longer needed, the structure does not allow for it.
```

Comment:
retain mention that "this requires indefinite amounts of disk space" due to the no-pruning